src/Anonymization/Anonymizer/AnonymizerRegistry.php

```diff
@@ -45,42 +51,66 @@ public function addPath(array $paths): void
     /**
      * Get all registered anonymizers classe names.
      *
-     * @return array<string,string>
-     *   Keys are names, values are class names.
+     * @return array<string,AsAnonymizer>
      */
-    public function getAnonymizers(): array
+    public function getAllAnonymizerMetadata(): array
     {
         $this->initialize();
 
-        return $this->anonymizers;
+        return $this->metadata;
     }
 
     /**
-     * Get anonymizer class name.
-     *
-     * @param string $name
-     *   Anonymizer name.
-     *
-     * @return string
-     *   Anonymizer class name.
+     * Create anonymizer instance.
      */
-    public function get(string $name): string
+    public function createAnonymizer(
+        string $name,
+        AnonymizerConfig $config,
+        Options $options,
```

Comment:
$options argument seems not to be used ? Why not remove it ?

Reply:
It is used actually, it's the last parameter of the anonymizer constructor, I might have accidentally forgotten it!